compiler/tools/fpp-check/test/component/tests.sh

```diff
@@ -12,6 +12,7 @@ enum_undef_constant
 enum_undef_type
 queued_no_async_input
 ok
+passive_statemachine
```

Comment:
I see active_statemachine.fpp but no test for it. Also let's be consistent: statemachine --> state_machine.

Reply:
Renamed to active_state_machine.fpp and passive_state_machine.fpp and added to tests.sh